Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/zsh
```

Comment:
Is zsh necessary for this to run? Because running this on VM failed because I don't have zsh. Can we have do it in a shell which is usually present?

Reply:
sorry, I will change all the scripts to bash.

Reply:
ennn, sorry, I did change all to bash, and resulted in lots of errors...
take some efforts to fix those error..
for hurry, I just changed them back to zsh...

Reply:
I would prefer to use zsh, for 2 reasons:
1: macos already changed the default shell to zsh.
2: zsh is more easier to customize, there are some wonderful open source plugins, like "oh-my-zsh"

Reply:
If we automate the build process, say for automation, we'll get a Linux or Windows machine, but not MacOS. It's fine for now.